src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,6 +111,19 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
+@include breakpoint($small-screen) {
```

Comment:
Just a reminder, we gotta change this to `$medium-screen`---thanks to your observation that almost no devices are within our small screen-width breakpoint these days 👍